Change default pr remote name from "fork" to "mine"

This is just my muscle memory. There's no way to change it otherwise.

--- cli-2.20.2.orig/pkg/cmd/pr/create/create.go
+++ cli-2.20.2/pkg/cmd/pr/create/create.go
@@ -726,7 +726,7 @@
 
 		// TODO: prevent clashes with another remote of a same name
 		gitClient := ctx.GitClient
-		gitRemote, err := gitClient.AddRemote(context.Background(), "fork", headRepoURL, []string{})
+		gitRemote, err := gitClient.AddRemote(context.Background(), "mine", headRepoURL, []string{})
 		if err != nil {
 			return fmt.Errorf("error adding remote: %w", err)
 		}
